backend/app/views/spree/admin/shared/_locale_selection.html.erb

```diff
@@ -1,7 +1,7 @@
-<% available_locales = Spree.i18n_available_locales %>
+<% available_locales = current_store.available_locales %>
```

Comment:
I think the `current_store.available_locales` should apply to the frontend only

Reply:
Ideally we would have two configurations, but this seems overkill to me. We could keep it as is, if we encourage the users of `solidus_i18n` to copy over only the files they need in their app (what I recommend anyway).

Reply:
Reverted